hw02/src/main/java/ru/otus/hw/homework/CustomerService.java

```diff
@@ -0,0 +1,30 @@
+package ru.otus.hw.homework;
+
+import java.util.Comparator;
+import java.util.Map;
+import java.util.Objects;
+import java.util.TreeMap;
+
+public class CustomerService {
+
+    private final TreeMap<Customer, String> map = new TreeMap<>(Comparator.comparingLong(Customer::getScores).thenComparingLong(Customer::getId));
```

Comment:
Лучше тип объявлять интерфейсом:
```suggestion
    private final NavigableMap<Customer, String> map = new TreeMap<>(Comparator.comparingLong(Customer::getScores).thenComparingLong(Customer::getId));
```